WORKSPACE

```diff
@@ -195,28 +188,29 @@ http_file(
 )
 
 # Docker rules.
-git_repository(
+http_archive(
     name = "io_bazel_rules_docker",
-    commit = "5eb0728594013d746959c4bd21aa4b0c3e3848d8",
-    remote = "https://github.com/bazelbuild/rules_docker.git",
+    sha256 = "aed1c249d4ec8f703edddf35cbe9dfaca0b5f5ea6e4cd9e83e99f3b0d1136c3d",
+    strip_prefix = "rules_docker-0.7.0",
+    urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
 )
 
+load("@io_bazel_rules_docker//container:container.bzl", "container_pull")
 load(
-    "@io_bazel_rules_docker//docker:docker.bzl",
-    "docker_pull",
-    "docker_repositories",
+    "@io_bazel_rules_docker//repositories:repositories.bzl",
+    container_repositories = "repositories",
 )
 
 # Used to generate java ca certs.
-docker_pull(
+container_pull(
     name = "debian8",
     # From tag: 2017-09-11-115552
     digest = "sha256:6d381d0bf292e31291136cff03b3209eb40ef6342fb790483fa1b9d3af84ae46",
     registry = "gcr.io",
     repository = "google-appengine/debian8",
 )
 
-docker_repositories()
+container_repositories()
```

Comment:
You probably want to move this to before the container_pull (and also load container_repositories before loading container_pull)

Reply:
Thanks, @nlopezgi. Re-ordered like below. Is this what you meant?

```diff
@@ -195,11 +195,13 @@ http_archive(
     urls = ["https://github.com/bazelbuild/rules_docker/archive/v0.7.0.tar.gz"],
 )
 
-load("@io_bazel_rules_docker//container:container.bzl", "container_pull")
 load(
     "@io_bazel_rules_docker//repositories:repositories.bzl",
     container_repositories = "repositories",
 )
+load("@io_bazel_rules_docker//container:container.bzl", "container_pull")
+
+container_repositories()
 
 # Used to generate java ca certs.
 container_pull(
@@ -210,8 +212,6 @@ container_pull(
     repository = "google-appengine/debian8",
 )
 
-container_repositories()
-
 # Have the py_image dependencies for testing.
 load(
     "@io_bazel_rules_docker//python:image.bzl",
```

Reply:
almost, the call to container_repositories should be before the load of container.bzl (to make sure any deps required by container.bzl are loaded)

Reply:
Ah, I see. Done.